src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -8560,7 +8560,7 @@ public static String substring(final String str, int start, int end) {
      * </pre>
      *
      * @param str  the String to get a substring from, may be null
-     * @param separator  the character to search.
+     * @param separator the character (Unicode code point) to search for
```

Comment:
Sentences should end in a period. Sentences feel quite awkward (to me) ending in a preposition.